plutus-pab/app/Main.hs

```diff
@@ -8,6 +8,7 @@
 {-# LANGUAGE MultiParamTypeClasses #-}
 {-# LANGUAGE NamedFieldPuns        #-}
 {-# LANGUAGE OverloadedStrings     #-}
+{-# LANGUAGE RecordWildCards       #-}
```

Comment:
Controversial :p I don't mind them... but maybe we need a style guide for the odd extensions we do object to.

Reply:
I think that would be a good idea, yes!